x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -467,10 +467,6 @@ public void forceMergeActionWithCodec(String codec) throws Exception {
         expectThrows(ResponseException.class, () -> indexDocument(client(), index));
     }
 
-    public void testForceMergeAction() throws Exception {
-        forceMergeActionWithCodec(null);
-    }
```

Comment:
This change is a little out of the scope of this PR, you did discover that we're missing a method below this one that has

```java
public void testForceMergeActionWithCompressionCodec() throws Exception {
    forceMergeActionWithCodec("best_compression");
}
```

So I think we should not apply this change in this PR

Reply:
+1, I'll back that out on this PR, and then open a new PR that adds the missing test.

Reply:
563720a for the revert

Reply:
https://github.com/elastic/elasticsearch/pull/64377 for the new test